Project-1/README.md

```diff
@@ -12,63 +12,296 @@ Tip #4: The work we do is challenging; that should be assumed. You are smart eno
+|  T |	T   |	F   |	T
+|  T |	T   |	T   |	T
+
+ (b) is contingent.
+ 
+| A  |	B   |	((A → B) ∧ (A → ¬B)) |
+|--- | ---  | -----------------------|
+| F  |	F   |	T                    |
+| F  |	T   |	T
+| T  |	F   |	F
+| T  |	T   |	F
+ 
+ (c) is a tautology
+ 
+| A  |	B   |	C   |	((A → (B ∨ C)) ∨ (C → ¬A)) |
+| -- | ---  | ----  | ---------------------------- |
+| F  |	F   |	F   |   T
+| F  |	F   |	T   |	T
+| F  |	T   |	F   |	T
+| F  |	T   |	T   |	T
+| T  |  F   |	F   |	T
+| T  |	F   |	T   |	T
+| T  |	T   |	F   |	T
+| T  |	T   |	T   |	T
+
+ (d) is contigent.
+ 
+| A  |	B   |	C   |	D   |	(((A → B) ∧ C) ∨ (A ∧ D)) |
+| -- | ---- | ----- | ----- | ----------------------------|
+| F  |	F   |	F   |	F   |	F
+| F  |	F   |	F   |	T   |	F
+| F  |	F   |	T   |	F   |	T
+| F  |	F   |	T   |	T   |	T
+| F  |	T   |   F   |	F   |	F
+| F  |	T   |	F   |	T   |	F
+| F  |	T   |	T   |	F   |	T
+| F  |	T   |	T   |	T   |	T
+| T  |	F   |	F   |	F   |	F
+| T  |	F   |	F   |	T   |	T
+| T  |	F   |	T   |	F   |	F
+| T  |	F   |	T   |	T   |	T
+| T  |	T   |	F   |	F   |	F
+| T  |	T   |	F   |	T   |	T
+| T  |	T   |	T   |	F   |	T
+| T  |	T   |	T   |	T   |	T
+
+
 	
 2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
-  ```
+  
   (a) (A→B)→C
   (b) (A→(B∨C))∨(C→¬A)
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
-  ```
+  
+  Answer:
+  
+  (a) CNF: (A∨C)∧(C∨(¬B))
+  
+  (b) CNF: C v ¬C
+  Since (b) is a tautology, its CNF form can be any formula of the form pv¬p. 
+  
+  (c) CNF: C v ¬C
+  (c) is a tautology.
   
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   
-  ```
-  (a)  B(x,y) that says that x is a brother of y  
-  (b)  A(x,y) that says that x is an aunt of y  
-  (c)  C(x,y) that says that x and y are cousins   
-  (d)  O(x) that says that x is an only child  
-  (e)  T(x) that says that x has exactly two brothers 
-  ```
+  (a)  B(x,y) ≡ ∃u(P(u,x)∧P(u,y))∧¬F(x)∧¬(x=y)
+
+  
+  (b)  A(x,y) ≡ ∃u∃w(P(u,y)∧P(w,x)∧P(w,u)∧¬(x=u))∧F(x)∧¬(x=y)
+  
+  
+  (c)  C(x,y) ≡ ∃u∃w∃z(P(w,z)∧P(w,u)∧P(z,x)∧P(u,y)∧¬(z=u))∧¬(x=y)
+  
+  
+  (d)  O(x) ≡ ∃u(P(u,x)∧∀y(P(u,y)→x=y))
+    
+ 
+  (e)  T(x) ≡ ∃u∃y∃z(¬(y=z)∧¬(x=y)∧¬(x=z)∧P(u,x)∧P(u,y)∧P(u,z)∧¬F(y)∧¬F(z))∧∀w(P(u,w)∧¬F(w)→ w=x ∨ w=y ∨ w=z)
+  
 
 4. Let V be a vocabulary of the attribute (concept) language with complements (ALC) consisting of a role name "parent_of" and a concept name "Male". Interpret parent_of as "x is a parent of y" and M as "x is male". Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   ```
   (a)  B that says that x is a brother of y
+
+  Answer: 
```

Comment:
Should be "ALC is not expressive enough"

Reply:
Thank you for pointing it out1